tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -684,3 +698,80 @@ def sync_volumes(volumes, index, vol_options, sync_ttl):
             b_index += 1
         except KeyError:
             break
+
+
+def brick_status_alert(hostname):
+    # fetching brick details of disconnected node
+    bricks = NS._int.client.read(
+        "clusters/%s/Bricks/all/%s" % (
+            NS.tendrl_context.integration_id,
+            hostname
+        )
+    )
+    for brick_info in bricks.leaves:
+        try:
+            brick_path = brick_info.key
+            lock = None
+            lock = etcd.Lock(
+                NS._int.client,
+                brick_path[1:]
```

Comment:
`brick_path[1:]` just to avoid initial `/` from path ??

Reply:
yes, if / came first then lock is not working, so i removed this